app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelContent.tsx

```diff
@@ -52,19 +52,27 @@ const ConnectionPanelContent = ({
 		icon = <DuplicateServerIcon />;
 	}
 
-	let secondaryButton;
+	let firstButton;
 
-	if (isIph) {
-		secondaryButton =
+	if (isButtonOneIph) {
+		firstButton =
+			firstButtonLabel &&
 			<InProductHelpAction
-				label="Learn more"
+				label={firstButtonLabel}
+				appearance={InProductHelpActionButtonAppearance.Primary}
 				type={InProductHelpActionType.HelpButton}
 				searchQuery={InProductHelpIds.PENDING_SERVER_LEARN_MORE}
 			/>;
 	} else {
-		secondaryButton =
-			buttonTwoOnClick &&
-			<Button onClick={() => buttonTwoOnClick?.(jenkinsServerUuid)}>{secondButtonLabel}</Button>;
+		firstButton =
```

Comment:
I see what you're doing here but there are a few changes we could make to simplify this.

1. As `firstButtonLabel` is being checked in both, I would move the check to the if statement condition. So now you have `if (isButtonOneIph && firstButtonLabel)`. Taking it one step further, you could put this check into it's own variable `const isInProductHelpAction = isButtonOneIph && firstButtonLabel` and call it from the if statement `if (isInProductHelpAction). This will make sense once I explain point 2.
2. I would move this check into the JSX. I would put it back on Line 85 (under `<ButtonGroup`>) and have something like this:
```
{
  isInProductHelpAction 
    ? <InProductHelpAction ...>
    : <Button ...>
}
```

This allows the reader to understand what's going on inline while they're reading the JSX. Let me know what you think :)

Reply:
sounds good. Move the check all to the render return